src/client/invoices.rs

```diff
@@ -42,6 +42,10 @@ pub struct Invoice {
     pub invoice_pdf: Option<String>,
     /// The total after any minimums, discounts, and taxes have been applied.
     pub total: String,
+    /// This is the final amount required to be charged to the
+    /// customer and reflects the application of the customer balance
+    /// to the total of the invoice.
+    pub amount_due: String,
```

Comment:
Hope the docs are right that this is never `null`.

Reply:
If they are I'd consider that a bug in Orb -- it logically doesn't make sense to have an invoice where we don't know how much we're charging someone